src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from "react-router-dom";
+
+function ArticleCard({ articles = [] }) {
```

Comment:
why are you defaulting it to an empty array here ? will it not be an empty array when declared using state ?